operator/bundle_helpers/README.md

```diff
@@ -38,19 +38,17 @@ chmod +x pip_find_builddeps.py
 
 ```bash
 ./pip_find_builddeps.py requirements.txt \
-  --append \
-  --only-write-on-update \
   -o requirements-build.in
 
-pip-compile requirements-build.in --allow-unsafe --generate-hashes > requirements-build.txt
+pip-compile requirements-build.in --allow-unsafe --generate-hashes
 ```
 
```

Comment:
A reminder to commit changes will be handy.

Reply:
done in https://github.com/stackrox/stackrox/pull/14257/commits/dbeb718219a34cb7575150de80d8fa323b21ebaf